outputs.tf

```diff
@@ -158,11 +158,93 @@ output "db_parameter_group_id" {
   value       = try(aws_db_parameter_group.this[0].id, null)
 }
 
-################################################################################
-# CloudWatch Log Group
-################################################################################
+#######################################################################################################################
+# RDS-PROXY: proxy output will show outputs other than empty string `""` , only when `create_db_proxy` is set to `true`
```

Comment:
what's this comment for?

Reply:
The RDS Proxy is an optional resource. The default value of `create_db_proxy` to create RDS Proxy is set to `false`
Hence the outputs for RDS Proxy will show values other than `null` or `" "` only when `create_db_proxy` is set to `true`